components/ResidentPage/CaseNoteDialog.spec.tsx

```diff
@@ -138,6 +141,33 @@ describe('CaseNoteDialog', () => {
     expect(screen.getAllByRole('definition').length).toBe(6);
   });
 
+  it('strips html tags from historic case notes', () => {
+    (useRouter as jest.Mock).mockReturnValueOnce({
+      query: {
+        case_note: mockedCaseNote.recordId,
+      },
+    });
+
+    render(
+      <CaseNoteDialog
+        totalCount={1}
+        socialCareId={123}
+        caseNotes={[
+          {
+            ...mockedCaseNote,
+            caseFormData: {
+              ...mockedCaseNote.caseFormData,
+              is_historical: true,
+            },
+          },
+        ]}
+      />
+    );
+
+    expect(screen.getByText('foo historic'));
+    expect(screen.queryByText('<h1>foo historic</h1>')).toBeNull;
```

Comment:
`.toBeNull()` ?

Reply:
Also just wondering if this actually works? Thinking that react will render it as `&lt;h1&gt;foo historic&lt;/h1&gt;` on the back?

Reply:
fixed typo